src/lib/i18n.ts

```diff
@@ -0,0 +1,86 @@
+import i18next from 'i18next';
+import { createI18nStore } from 'svelte-i18next';
+
+i18next.init({
+	lng: 'en',
+	resources: {
+		en: {
+			translation: {
+				nav: {
+					sessions: 'Sessions',
+					knowledge: 'Knowledge',
+					settings: 'Settings',
+					motd: 'Motd',
+					theme: { dark: 'Dark', light: 'Light' }
+				},
+				sessions: {
+					pageTitle: 'Sessions',
+					new: 'New session',
+					empty: 'No sessions',
+					session: 'Session',
```

Comment:
Let's get rid of all the `pageTitle`'s.

Instead of `pageTitle: Sessions` and `session: Session` let's use the [pluralization syntax](https://www.i18next.com/translation-function/plurals):

```
one: Session
other: Sessions
```

```
<Head title={$i18n.t('sessions', count: 0)} />
```